tests/unit/DateUtilsTest.js

```diff
@@ -82,57 +96,59 @@ describe('DateUtils', () => {
     });
 
     it('canUpdateTimezone should return true when lastUpdatedTimezoneTime is more than 5 minutes ago', () => {
-        const currentTime = moment().add(6, 'minutes');
-        moment.now = jest.fn(() => currentTime);
+        // Use fake timers to control the current time
+        jest.useFakeTimers('modern');
+        jest.setSystemTime(addMinutes(new Date(), 6));
         const isUpdateTimezoneAllowed = DateUtils.canUpdateTimezone();
         expect(isUpdateTimezoneAllowed).toBe(true);
     });
 
     it('canUpdateTimezone should return false when lastUpdatedTimezoneTime is less than 5 minutes ago', () => {
-        const currentTime = moment().add(4, 'minutes');
-        moment.now = jest.fn(() => currentTime);
+        // Use fake timers to control the current time
+        jest.useFakeTimers('modern');
+        jest.setSystemTime(addMinutes(new Date(), 4));
         const isUpdateTimezoneAllowed = DateUtils.canUpdateTimezone();
         expect(isUpdateTimezoneAllowed).toBe(false);
     });
 
     it('should return the date in calendar time when calling datetimeToRelative', () => {
-        const aFewSecondsAgo = moment().subtract(10, 'seconds');
-        expect(DateUtils.datetimeToRelative(LOCALE, aFewSecondsAgo)).toBe('a few seconds ago');
+        const aFewSecondsAgo = subSeconds(new Date(), 10);
+        expect(DateUtils.datetimeToRelative(LOCALE, aFewSecondsAgo)).toBe('less than a minute');
 
-        const aMinuteAgo = moment().subtract(1, 'minute');
-        expect(DateUtils.datetimeToRelative(LOCALE, aMinuteAgo)).toBe('a minute ago');
+        const aMinuteAgo = subMinutes(new Date(), 1);
+        expect(DateUtils.datetimeToRelative(LOCALE, aMinuteAgo)).toBe('1 minute');
 
-        const anHourAgo = moment().subtract(1, 'hour');
-        expect(DateUtils.datetimeToRelative(LOCALE, anHourAgo)).toBe('an hour ago');
+        const anHourAgo = subHours(new Date(), 1);
+        expect(DateUtils.datetimeToRelative(LOCALE, anHourAgo)).toBe('about 1 hour');
```

Comment:
Is this behavior change expected? Can't this cause impact in what we see in the App?

Reply:
Good one, need to check the usages to see if this does not break an outer string or something